beaglebone/imu/src/__main__.py

```diff
@@ -184,8 +364,14 @@ def main():
         for reg in l:
             manager.turnOffRegister(reg)
 
+        manager.get_calibration_vals(manager.calibration_matrix, manager.mag_offsets)
+        while(True):
```

Comment:
confused why this while loop is here? has this ever published to LCM? the object/run coroutines isn't called until after